### PR TITLE
Created models, routes, controllers, and form, connection with firebase.

### PR DESCRIPTION
**Pull request by Easiv**

### CODE REVIEW

File: app/controllers/books.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Controller from '@ember/controller';
```
Comment (PoslinskiNet) [inline app/controllers/books.js]: ✂️

File: app/controllers/books/new.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  actions: {
+    createBook: function() {
```
Comment (PoslinskiNet) [inline app/controllers/books/new.js]: ESLint use `ember-suave` & `eslint:recommended` in .eslintrc.js

Comment (PoslinskiNet) [inline app/controllers/books/new.js]: Use new shorthand syntax for object function members.

File: app/models/book.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  author: DS.attr('string'),
+  //creationDate: DS.attr('date'),
```
Comment (PoslinskiNet) [inline app/models/book.js]: ✂️

File: app/templates/application.hbs [hunk 4]
```diff
@@ -20,6 +20,11 @@
         <li class="nav-item">
           <a class="nav-link"{{action "switchPage"}}>{{#link-to 'page.new'}} {{t 'write.text'}} {{/link-to}}</a>
         </li>
+        {{#if session.isAuthenticated}}
+        <li class="nav-item">
```
Comment (PoslinskiNet) [inline app/templates/application.hbs]: Add some indentation after conditionals.

File: app/templates/books/new.hbs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{{outlet}}
+
+<h2 id="newBook">New Book</h2>
```
Comment (PoslinskiNet) [inline app/templates/books/new.hbs]: Translation

File: app/templates/books/new.hbs [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+<form>
+  <div class="form-group">
+    <label for="titleInput">Title</label>
+    {{input value=title placeholder="Title" class="form-control" id="titleInput"}}
```
Comment (PoslinskiNet) [inline app/templates/books/new.hbs]: Translation in placeholder, the same in author

File: app/adapters/application.js [hunk 7]
```diff
@@ -1,4 +1,6 @@
 import FirebaseAdapter from 'emberfire/adapters/firebase';
+import { inject } from '@ember/service';
 
 export default FirebaseAdapter.extend({
+  firebase: inject()
```
Comment (PoslinskiNet) [inline app/adapters/application.js]: Try to remove it - I think it is redundant.

**Comment by PoslinskiNet** [comment]
Provide description of what this code does.

File: app/controllers/books.js (hunk 1 above)
Comment (Easiv) [reply app/controllers/books.js]: Deleting it causes disappearing half of the content in the app.

Comment (PoslinskiNet) [reply app/controllers/books.js]: ✂️, in that case, means - remove this file completely.

**Pull request merged**